selenium-examples/src/test/java/com/saucedemo/SampleHeadlessSauceTest.java

```diff
@@ -1,3 +1,5 @@
+package com.saucedemo;
```

Comment:
@coleve27 also, I know it's not your fault, but this example should use [Sauce Bindings](https://github.com/saucelabs/sauce_bindings). So you should do something like this:

```java
        SauceOptions options = new SauceOptions();
        options.setPlatformName(SaucePlatform.LINUX);

        SauceSession session = new SauceSession(options);
        session.setDataCenter(DataCenter.US_EAST);
        
        driver = session.start();
```
This will allow you to replace pretty much all the code that's currently there. Sauce Bindings is already part of the root POM.xml so it doesn't need to be added to this POM